system/modules/core/forms/FormFileUpload.php

```diff
@@ -238,6 +250,8 @@ public function validate()
 
 					// Generate the DB entries
 					$strFile = $strUploadFolder . '/' . $file['name'];
+
+					/** @var \FilesModel $objFile */
```

Comment:
Hmm, could we use `@return \Model\Collection|static` for all collections? Because the API for would be identical and available?